apm-agent-api/src/main/java/co/elastic/apm/api/ElasticApm.java

```diff
@@ -232,4 +232,32 @@ public static void captureException(@Nullable Throwable e) {
         // co.elastic.apm.api.ElasticApmInstrumentation.CaptureExceptionInstrumentation.captureException
     }
 
+    /**
+     * Gets the service name and version that is associated with a class loader
+     *
+     * @param classLoader the class loader for which the service name and version should be looked up
+     * @return the service name and version for the given class loader null
+     */
+    @Nullable
+    public static ServiceInfo getServiceInfoForClassLoader(@Nullable ClassLoader classLoader) {
```

Comment:
Get the service name and version through separate APIs that return a `String`, or just remove (I prefer the latter).

Reply:
Done